Rakefile

```diff
@@ -1,5 +1,154 @@
 require "bundler"
 Bundler.require
+require 'smarter_csv'
 
 require "sinatra/activerecord/rake"
 
+require File.expand_path('../config/environment', __FILE__)
+
+
+namespace :import do
+  desc "Import station.csv to cities table"
+  task :cities do
+   create_cities
+  end
+
+  desc "Import station.csv to table"
+  task :stations do
+   create_stations
+  end
+
+  desc "Import station.csv to subscription_types table"
+  task :subscription_types do
+   create_subscription_types
+  end
+
+  desc "Import weather.csv to table"
+  task :weather_conditions do
+    create_weather_conditions
+  end
+
+  desc "Import trip.csv to table"
+  task :trips do
+    create_trips
+  end
+
+end
+
```

Comment:
there's a lot going on in each of the methods below. could we create a module to help us parse and clean our data? that way, the behavior will be encapsulated in a module and we can reuse it if necessary